.gitignore

```diff
@@ -1,6 +1,42 @@
+# Node modules
 node_modules/
+
+# Dependency files
+package.json
+yarn.lock
+package-lock.json
+
+# Config files
+config.json
```

Comment:
this must be included, otherwise Fredy will break for everybody when cloning freshly